### PR TITLE
chore: bump nodejs version to fix security issues

### LINKED ISSUE #662: NodeJS image is released with critical NodeJS vulnerability
I just scanned `docker-image|gcr.io/distroless/nodejs-debian10` with Snyk, and find that the nodeJS version is outdated:

```

✗ Medium severity vulnerability found in node
  Description: Buffer Overflow
  Info: https://snyk.io/vuln/SNYK-UPSTREAM-NODE-1009920
  Introduced through: node@14.6.0
  From: node@14.6.0
  Fixed in: 14.9.0

✗ Medium severity vulnerability found in node
  Description: HTTP Request Smuggling
  Info: https://snyk.io/vuln/SNYK-UPSTREAM-NODE-674660
  Introduced through: node@14.6.0
  From: node@14.6.0
  Fixed in: 14.11.0

✗ High severity vulnerability found in node
  Description: Denial of Service (DoS)
  Info: https://snyk.io/vuln/SNYK-UPSTREAM-NODE-674659
  Introduced through: node@14.6.0
  From: node@14.6.0
  Fixed in: 14.11.0
```

Looking at the Bazel source file, I guess solving this is just re-build of the image to catch the latest version.

Can you please help with bumping a minor version of node?

### PR DESCRIPTION
**Pull request by omerlh**
close #662 

I've spent some time trying to test it locally, but Bazel kept failing on random errors...

### CODE REVIEW

File: WORKSPACE [hunk 1]
```diff
@@ -311,55 +311,55 @@ http_archive(
 http_archive(
     name = "nodejs10_amd64",
     build_file = "//nodejs:BUILD.nodejs",
-    sha256 = "aa7e9e1d8abcc169119bf5c56ede515689f2644ccc4d40ca0fc33756a3deb1f7",
+    sha256 = "19cccb78f0881a78051291a50200200a0303649ee84e5489c771d3b4e4bd0e51",
     strip_prefix = "node-v10.22.0-linux-x64/",
```
Comment (chanseokoh) [inline WORKSPACE:315]: You should fix this line too. (It is the top-level directory name to strip for the files in the tarball.)

**Review by chanseokoh — COMMENTED** [review]
Thank you for your contribution!

File: WORKSPACE (hunk 1 above)
Comment (omerlh) [reply WORKSPACE]: fixed, thanks for pointing it out!

**Comment by chanseokoh** [comment]
And could you follow the CLA bot instructions above? We cannot accept your contribution until you agree and sign it.

**Comment by omerlh** [comment]
I am waiting for out IT to create the required group, I am not sure when this will happen with the holiday season :(

File: WORKSPACE [hunk 2]
```diff
@@ -311,55 +311,55 @@ http_archive(
 http_archive(
     name = "nodejs10_amd64",
     build_file = "//nodejs:BUILD.nodejs",
-    sha256 = "aa7e9e1d8abcc169119bf5c56ede515689f2644ccc4d40ca0fc33756a3deb1f7",
-    strip_prefix = "node-v10.22.0-linux-x64/",
+    sha256 = "19cccb78f0881a78051291a50200200a0303649ee84e5489c771d3b4e4bd0e51",
+    strip_prefix = "node-v10.23.0-linux-x64/",
     type = "tar.gz",
-    urls = ["https://nodejs.org/dist/v10.22.0/node-v10.22.0-linux-x64.tar.gz"],
+    urls = ["https://nodejs.org/dist/v10.22.0/node-v10.23.0-linux-x64.tar.gz"],
```
Comment (chanseokoh) [inline WORKSPACE:317]: Probably
```suggestion
    urls = ["https://nodejs.org/dist/v10.23.0/node-v10.23.0-linux-x64.tar.gz"],
```
Please fix other URLs too.

Comment (omerlh) [reply WORKSPACE]: oops, I fixed other places also - thanks!

**Comment by omerlh** [comment]
@googlebot I signed it!

**Comment by omerlh** [comment]
@googlebot I signed it!

**Comment by omerlh** [comment]
@chanseokoh do you know who can help me with the CLA?

**Comment by omerlh** [comment]
Another day another NodeJS security release - just updated to include the latest security fix

**Comment by chanseokoh** [comment]
Are you an individual signer or corporate signer? What does the CLA check page says?

> * Check [your existing CLA data](https://cla.developers.google.com/clas) and verify that your [email is set on your git commits](https://help.github.com/articles/setting-your-email-in-git/).

**Comment by omerlh** [comment]
I'm corporate signer, and I singed the thing on doco sign - this is the weird thing! There is nothing on this link :(

**Comment by chanseokoh** [comment]
Just in case, could you ping the CLA bot again? I do see that "Snyk Ltd" has a corporate CLA agreement in the list. (I may be wrong, but I feel I did not see it before.)

**Comment by omerlh** [comment]
@googlebot I signed it!

**Comment by chanseokoh** [comment]
You might have read it already, but I think the [troubleshooting section](https://opensource.google/docs/cla/#troubleshoot) is the best resources you can use. At least I can confirm "Snyk Ltd" has a corporate CLA agreement. Perhaps the email covered by your CLA agreement is different from the email you're using for Git commits or is not associated to your GiHub account?

**Comment by omerlh** [comment]
> If the contributor is covered by a corporate CLA, make sure that the corporation is listed at http://linkremoved/.

Where can I check the the corporate is listed? I am pretty sure I am using the right email...

**Comment by chanseokoh** [comment]
The link is an internal one, so that's why you're seeing "linkremoved" in the public version of the doc. I'll try to get some help, but also double-check that your email is also correctly added to your corporate CLA's authorized contributor group.

**Comment by loosebazooka** [comment]
I don't know if this helps, but for corp contributors:
```
How do I add authorized contributors for my company?
First, add their email address to the Google Group that you submitted with the corporate CLA. This is most commonly their company address.
```

**Comment by chanseokoh** [comment]
@omerlh I got the response from the CLA team. Make sure your email is in that group.

> Omer is not a member of google-cla-users@snyk.io. Email addresses of covered users need to be in that group.

**Comment by omerlh** [comment]
@googlebot I signed it!

**Comment by omerlh** [comment]
YESS thank you so much! The confusion was in the fact we already had a group that I did not know about. Now it is finally signed!

**Review by chanseokoh — APPROVED** [review]

**Comment by chanseokoh** [comment]
Great, thank you for upgrading these. Images will go live within an hour.

**Pull request merged**